backoffice/templates/backoffice/user.html

```diff
@@ -162,6 +155,16 @@ <h3>{% trans "Enrollments" %}</h3>
 
         <div class="col-md-12">
 
+            {% if userform.instance.is_active and not userform.instance.is_superuser and not userform.instance.is_staff %}
+                <form action="{% url 'masquerade-mask' %}" method="POST" target="_blank" style="display: inline">
+                    {% csrf_token %}
+                    <input type="hidden" name="mask_user" value="{{ userform.instance.username }}">
+                    <button class="btn btn-info" type="submit">{% trans "Impersonate" %}</button>
+                </form>
+
+                <a class="btn btn-warning" onclick="$.get({% url 'masquerade-unmask' %});">{% trans "Stop impersonation" %}</a>
```

Comment:
Oo I would expect this view to only accept `POST` HTTP verb, but it's not...

Reply:
because it's to stop impersonating so no risk at all.

Reply:
This view modifies user's session, so IMO allowing `GET` is not a good design. I'm talking of the implementation of this view not the way you use it.

Reply:
True